### ISSUE TITLE
installer fails on Mac using ansible 2.4 

### ISSUE THREAD

**Issue by ffirg** [ansible/awx#255]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - Installer

##### SUMMARY
Installer barfs when using ansible 2.4 on Mac (installed using pip) 

```
$ ansible-playbook -i inventory install.yml -e openshift_password=developer -e docker_registry_password=$(oc whoami -t) -vvv
ansible-playbook 2.4.0.0
  config file = None
  configured module search path = [u'/usr/local/lib/python2.7/site-packages/ara/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible-playbook
  python version = 2.7.13 (default, Feb 21 2017, 12:15:23) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
No config file found; using defaults
Parsed /Users/pgriffit/awx/awx/installer/inventory inventory source with ini plugin
[DEPRECATION WARNING]: The use of 'include' for tasks has been deprecated. Use 'import_tasks' for static inclusions or 'include_tasks' for dynamic inclusions.
This feature will be removed in a future release. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: include is kept for backwards compatibility but usage is discouraged. The module documentation details page may explain more about this
rationale.. This feature will be removed in a future release. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
statically imported: /Users/pgriffit/awx/awx/installer/check_vars/tasks/check_openshift.yml
statically imported: /Users/pgriffit/awx/awx/installer/check_vars/tasks/check_docker.yml
 [WARNING]: While constructing a mapping from /Users/pgriffit/awx/awx/installer/local_docker/tasks/main.yml, line 207, column 7, found a duplicate dict key
(SECRET_KEY). Using last defined value only.

ERROR! Unexpected Exception, this is probably a bug: cannot import name load_config_file
the full traceback was:

Traceback (most recent call last):
  File "/usr/local/bin/ansible-playbook", line 106, in <module>
    exit_code = cli.run()
  File "/usr/local/lib/python2.7/site-packages/ansible/cli/playbook.py", line 130, in run
    results = pbex.run()
  File "/usr/local/lib/python2.7/site-packages/ansible/executor/playbook_executor.py", line 90, in run
    self._tqm.load_callbacks()
  File "/usr/local/lib/python2.7/site-packages/ansible/executor/task_queue_manager.py", line 184, in load_callbacks
    for callback_plugin in callback_loader.all(class_only=True):
  File "/usr/local/lib/python2.7/site-packages/ansible/plugins/loader.py", line 426, in all
    self._module_cache[path] = self._load_module_source(name, path)
  File "/usr/local/lib/python2.7/site-packages/ansible/plugins/loader.py", line 339, in _load_module_source
    module = imp.load_source(full_name, path, module_file)
  File "/usr/local/lib/python2.7/site-packages/ara/plugins/callbacks/log_ara.py", line 29, in <module>
    from ara.webapp import create_app
  File "/usr/local/lib/python2.7/site-packages/ara/webapp.py", line 15, in <module>
    import ara.config
  File "/usr/local/lib/python2.7/site-packages/ara/config.py", line 25, in <module>
    from ansible.constants import load_config_file
ImportError: cannot import name load_config_file
```

##### ENVIRONMENT
MacOS Sierra 10.12.6

$ pip show ansible
Name: ansible
Version: 2.4.0.0
Summary: Radically simple IT automation
Home-page: https://ansible.com/
Author: Ansible, Inc.
Author-email: info@ansible.com
License: GPLv3+
Location: /usr/local/lib/python2.7/site-packages
Requires: paramiko, cryptography, setuptools, jinja2, PyYAML

##### ACTUAL RESULTS

see output above

##### ADDITIONAL INFORMATION

Installer runs when downgrade back to 2.3
$ pip install ansible==2.3.2.0
$ ansible --version
ansible 2.3.2.0
  config file =
  configured module search path = [u'/usr/local/lib/python2.7/site-packages/ara/plugins/modules']
  python version = 2.7.13 (default, Feb 21 2017, 12:15:23) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]

**Comment by matburt** [comment]
That looks like an ARA problem to me. I don't think ARA supports Ansible 2.4 yet. Please try removing that from your module search path and trying again.

**Comment by dmsimard** [comment]
@ffirg hey there, @matburt pinged me about this. ARA 0.14.4 with Ansible 2.4 support was *just* released, version is 0.14.4. It should be available on PyPi shortly.

**Comment by ffirg** [comment]
I removed the ara python module and it did indeed get past this error. Thanks.
